Decred Wallet/Assets.xcassets/nav_menu/ic_accounts.imageset/Contents.json

```diff
@@ -2,14 +2,17 @@
   "images" : [
     {
       "idiom" : "universal",
+      "filename" : "accounts.png",
```

Comment:
How does this differ from the account imageset?

Reply:
the account imagesets used the icons from the older iteration. it has been removed and the current `ic_wallet01_24px{@nx}` renamed to `ic_wallets@{nx}.png` icon from the recent mockup used in commit 37e2453